### ISSUE TITLE
feature request: auth by fingerprint

### ISSUE THREAD

**Issue by zeliboba7** [ftsdesign/paranoid-diary#16]
In addition to password authentication it would be very convenient to use fingerprint, since it is supported by many recent phones.

**Comment by ftsdesign** [comment]
Good point, but probably will need to be optional and disabled by default, for security.
